homeassistant/components/thread/discovery.py

```diff
@@ -57,25 +57,32 @@ def try_decode(value: bytes | None) -> str | None:
         except UnicodeDecodeError:
             return None
 
-    ext_addr = service.properties.get(b"xa")
-    ext_pan_id = service.properties.get(b"xp")
-    network_name = try_decode(service.properties.get(b"nn"))
-    model_name = try_decode(service.properties.get(b"mn"))
+    service_properties = service.properties
```

Comment:
This was typed to `dict` before in upstream `zeroconf` so `mypy` never errored before